dashboards/dashboards.libsonnet

```diff
@@ -709,6 +709,84 @@ local tsLegend = tsOptions.legend;
       ) +
       row.withRepeat('instance'),
 
+    // Set this to 0 for the flat layout, -9 for the collapsed layout
+    local yOffset = if $._config.summaryRowCollapsed then -9 else 0,
+
+    local individualProbes = [
```

Comment:
If u can move the list start `[` to the next line I think the linter will pass the changes.

Reply:
Also just run make.